connectors/tests/test_byoc.py

```diff
@@ -1056,6 +1071,21 @@ def test_advanced_rules_present(filtering, should_advanced_rules_be_present):
     assert Filter(filtering).has_advanced_rules() == should_advanced_rules_be_present
 
 
+@pytest.mark.parametrize(
+    "filtering, validation_state, has_expected_validation_state",
+    [
+        (FILTER_VALIDATION_STATE_VALID, FilteringValidationState.EDITED, False),
+        (FILTER_VALIDATION_STATE_INVALID, FilteringValidationState.EDITED, False),
+        (FILTER_VALIDATION_STATE_EDITED, FilteringValidationState.EDITED, True),
+    ],
+)
+def test_is_in_state_edited(filtering, validation_state, has_expected_validation_state):
```

Comment:
probably should rename it to `test_has_validation_state`

Reply:
Yes, good catch! Forgot to rename it